src/SpotifyApi.NetCore.Tests/FollowApiTests.cs

```diff
@@ -11,87 +11,64 @@ namespace SpotifyApi.NetCore.Tests
     [TestClass]
     public class FollowApiTests
     {
-        [TestCategory("Integration")]
-        [TestMethod]
-        public async Task CheckCurrentUserFollowsArtists_ArtistId_AnyItems()
+        FollowApi api;
+        UsersProfileApi usersApi;
+        string bearerAccessToken;
+
+        public FollowApiTests()
         {
-            // arrange
             var http = new HttpClient();
             IConfiguration testConfig = TestsHelper.GetLocalConfig();
+            bearerAccessToken = testConfig.GetValue(typeof(string), 
+                "SpotifyUserBearerAccessToken").ToString();
             var accounts = new AccountsService(http, testConfig);
+            api = new FollowApi(http, accounts);
+            usersApi = new UsersProfileApi(http, accounts);
+        }
```

Comment:
Reusing objects in test classes can have unintended consequences. If you want to keep your test code clean, put this code into a method called `Initialize()` with a `[TestInitialize]` attribute. This means the objects get re-initialized on every test, but you can keep your code clean.